### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_ScreenReader:The labels are not present for the edit box.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9339]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudio;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.1.1;#MAS3.3.2;#NVDA;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Screen reader:NVDA, JAWS
MAS References:MAS1.1.1 MAS3.3.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Launch screen reader as NVDA
8. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
9. Navigate to the 9 fields using tab under "advanced storage setting button"

**Actual:**
When navigated to all 9 fields under advanced storage setting button there is no label provided for the fields, just a placeholder that is similar for all and announced by a screen reader.
**Except for the first field of every row.**

Issue observed with JAWS

Expected:
Each input field should have a label or instruction and the label should help the user complete the task.

**User Impact:**
Screen reader and keyboard users will not know with respect to what label the data needs to be entered in the field.

**Attachment link for Reference :**
https://microsoft-my.sharepoint.com/:p:/g/personal/v-stfe_microsoft_com1/EZsjFrfTsJJLnUL-OeO8oMUBBIykKGuh20brFBZSvIcT5A?e=FZLWgN

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/azuredatastudio/issues/9345

**Comment by Yashi1201** [comment]
As "SQL Server Big Data Cluster" feature has been removed
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
